website/docs/getting-started/setup/installation-guides/restack.md

```diff
@@ -0,0 +1,28 @@
+# Restack
+
+ [Restack](https://www.restack.io) is platform to discover and deploy production ready open source tools in minutes.
+ You only need to bring your AWS account and within minutes, your favorite open source tool would be deployed on Kubernetes, on your own cluster.
+ Restack is made so that developers can concentrate on coding and don’t have to worry about DevOps. 
+ Restack provides auto scalable, cloud-native, instant, production-ready deployment of open source tools.
+
+ ## One-click deployment
+
+ You can deploy Appsmith on the Restack platform with just one click.
+
+ The easiest way to do this is
+
+ * [Login to Restack](https://restack.io/login) with your GitHub or email.
+ * Connect your AWS account to create a cluster. 
+ * Search for Appsmith in the restack console and click to trigger the deployment. 
+
+You could also follow the detailed guide for [deploying Appsmith](https://www.restack.io/docs/deploy-appsmith-on-restack#overview) available on Restack docs.
```

Comment:
@jnikhila We can add this active link - https://www.restack.io/docs/deploy-appsmith-on-kubernetes and close the PR

Reply:
Done.